tests/end_to_end/server_test.py

```diff
@@ -40,7 +39,6 @@ def run(self, *args, **kwargs):
             self.server.start()
             super(ServerTest, self).run(*args, **kwargs)
         except Exception as e:
-            self.server.printDebugInfo()
```

Comment:
I agree with dcolligan here -- this does no harm in my view. If it's leading to spam, we should make an issue.